src/DynamoCoreWpf/ViewModels/Core/WorkspaceViewModel.cs

```diff
@@ -454,9 +456,11 @@ public WorkspaceViewModel(WorkspaceModel model, DynamoViewModel dynamoViewModel)
             Model.ConnectorAdded += Connectors_ConnectorAdded;
             Model.ConnectorDeleted += Connectors_ConnectorDeleted;
             Model.PropertyChanged += ModelPropertyChanged;
+            
+            this.refreshViewHandler = (sender, e) => { RefreshViewOnSelectionChange(); };
```

Comment:
Maybe this should be a method instead of a lambda expression for clarity/continuity?